swift/Sources/FlatBuffers/FlatBufferBuilder.swift

```diff
@@ -219,7 +219,7 @@ extension FlatBufferBuilder {
     ///   - bufSize: Current size of the buffer + the offset of the object to be written
     ///   - elementSize: Element size
     fileprivate func padding(bufSize: UInt32, elementSize: UInt32) -> UInt32 {
-        ((~bufSize) + 1) & (elementSize - 1)
+        (bufSize == 0 ? 0 : ((~bufSize) + 1)) & (elementSize - 1)
```

Comment:
Why is this needed? This should result in 0 even without the conditional.

Note C++ version here: https://github.com/google/flatbuffers/blob/89418eb848945d6e833698153a7a13f5b53e110b/include/flatbuffers/base.h#L393-L395

Reply:
well, swift doesn't have `__supress_ubsan__("unsigned-integer-overflow")`, however I found out we can achieve the same behavior if we use value overflow 
`https://docs.swift.org/swift-book/LanguageGuide/AdvancedOperators.html`
@vkill please take a look at the link

Reply:
@mustiikhalil It has been completed.

Reply:
@vkill nice! @aardappel what do you think?

Reply:
Wow, that's a Swift feature? Impressive.. I guess I shouldn't be surprised, since it was designed by @lattner :)